projects/wc/src/app/components/generic-ui/detail-view/detail-view.component.ts

```diff
@@ -83,9 +83,7 @@ export class DetailViewComponent {
         this.context().resourceDefinition.ui?.detailView?.fields ||
         defaultFields;
       this.resourceDefinition = this.context().resourceDefinition;
-      const url = window.location.pathname;
-      const namespaceMatch = url.match(/\/namespaces\/([^\/]+)/);
-      this.namespace = namespaceMatch ? namespaceMatch[1] : this.LuigiClient().getNodeParams(true)?.['namespace'];
+      this.namespace = this.context().namespaceId;
```

Comment:
if fact you don't need a separate field for keeping this value just add this to the` this.resourceService.read(`